server/channels/db/migrations/mysql/000110_delete_empty_drafts.up.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE Drafts
+SET DeleteAt = UpdateAt WHERE Message = ''
```

Comment:
Out of curiosity, why do we keep rows at all when deleting a draft? What's the use of a soft deleted empty message?

Reply:
This I'm not sure, I assume it could be for an audit trail? We don't have an actual `DELETE` function in the SQL store file, so I didn't want to add one and go too far outside the current spec.

Reply:
I'm wondering if we need to clarify the spec here. As it stands, this table has a theoretical max size equivalent to  `ChannelMembers` if we aren't cleaning up drafts (imagine every user typing then undoing in all their channels, dms, etc. exactly once over an organic period of time [like me, apparently ;P]). It feels like maybe just an oversight in the original plan?

Reply:
Maybe include drafts in data-retention? But anyways, beyond the scope of this PR.

Reply:
Yeah I'd like to keep the scope to at least just making sure the drafts aren't leaking through the API. I think @lieut-data does make a good point that it's worth investigating if we need to keep the rows or not.

Reply:
I'd like to pushback on keeping the scope to the the first half of the problem -- we're already having to do a data migration here, and it would be a shame to do another one to delete the empty rows just because we split the work in two artificially.

Reply:
Okay, let's have a discussion in the ICU channel about this. We'll get everyone's opinion involved and see what the best direction is :)

Reply:
I agree with Jesse here. There is no value in preserving empty rows in the DB that I can see.

Reply:
Doesn't seem like anyone else disagrees so I will move to actually deleting the empty rows.

Reply:
@lieut-data Made changes if you'd like to look again